content/03-tea/05-breakdown-view.md

```diff
@@ -3,16 +3,16 @@ title       = "View"
 weight      = 5
 +++
 
-The next part is the view:
+The next part is the `view` function:
 
 ```elm
 view : Model -> Html Msg
 view model =
     div [] [ text model ]
 ```
 
-This is a function that take a `Model` and return `Html`.
+This is a function that takes a `Model` argument and returns `Html`.
 
-Note `Html Msg`. `Msg` is a type variable. This means that this `Html` emits messages of a concrete type `Msg`. 
+If you’re wondering about the `Html Msg`. The `Msg` here is called a type variable. In this example, it indicates that the `Html` that our `view` returns  will only ever emit the messages specified in our program’s `Msg` type.  Being specific about this enables the Elm compiler to guarantee that our program handles all possible messages.
```

Comment:
I put `type variable` initially here. But this is not correct. A type variable would `a` in be `Html a`.
`Msg` in `Html Msg` is something else. A concrete type?

Maybe let's write:

A little explanation about `Html Msg`. If you see `Html msg`, the lowercase `msg` is a type variable. Meaning that `Html` can be of any type. Like generic in other languages (e.g. Html<T>).

If you see `Html Msg` it means that this type have a concrete type. In this example, it indicates that the `Html` that our `view` returns  will only ever emit the messages specified in our program’s `Msg` type.